sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UseUriInsteadOfString.cs

```diff
@@ -44,7 +50,8 @@ public sealed class UseUriInsteadOfString : SonarDiagnosticAnalyzer
         private static readonly DiagnosticDescriptor rule_S3996 =
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId_RuleS3996, MessageFormat_RuleS3996, RspecStrings.ResourceManager);
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(rule_S3995, rule_S3996);
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
+            => ImmutableArray.Create(rule_S3994, rule_S3995, rule_S3996);
 
         private static readonly HashSet<string> UrlNameVariants = new HashSet<string> { "uri", "url", "urn" };
```

Comment:
No real benefits to be honest. That's more a 'usage' but I am fine if you keep it like this.